### ISSUE TITLE
Unable to import from Tandoor

### ISSUE THREAD

**Issue by davidszerman** [YunoHost-Apps/tandoor_ynh#7]
Hello, 

i had a Tandoor installation via Docker and then installed via Yunohost

I made an export (zip file) from the app on Docker and then Import to the Yunohost version.

The import process is able to read the file (I get the list of recipes) but the import fails "unable to create resource"

Thank you for your help

**Comment by navanchauhan** [comment]
Hi

If it is not too much to ask, can you upload the zip file so I can test it out

**Comment by davidszerman** [comment]
Hello,

here is the file with one recipe ; I tested to reimport the file into the source app (docker version) and it worked so it is not related to the exported file

[export_2022-08-17.zip](https://github.com/YunoHost-Apps/tandoor_ynh/files/9357516/export_2022-08-17.zip)

**Comment by navanchauhan** [comment]
Thank you for this, I am investigating this issue now

**Comment by navanchauhan** [comment]
I am able to recreate the issue

**Comment by davidszerman** [comment]
Ho

for your information, I tried the bookmarklet in order to add a recipe. Same error message

**Comment by navanchauhan** [comment]
> Ho
> 
> 
> 
> for your information, I tried the bookmarklet in order to add a recipe. Same error message

Thank you for trying that out. I'm in a bind right now because when I enabled DEBUG and SQL_DEBUG I was able to import the recipes. So there's a problem somewhere which I need to figure out

**Comment by florisre** [comment]
Hi! I am experiencing the same issue. How can I help?

**Comment by navanchauhan** [comment]
Please reopen if the issue still persists. The fix has been merged into the master branch

**Comment by davidszerman** [comment]
It works like a charm.

Thanks !

**Comment by florisre** [comment]
I am still running into this issue, even after reinstalling Tandoor. Any chance YunoHost does not install the main branch by default?

**Comment by florisre** [comment]
> Please reopen if the issue still persists. The fix has been merged into the master branch

Still not working, don't know how to reopen.

**Comment by navanchauhan** [comment]
Alright, the latest package should have debug mode enabled to collect more data

**Comment by florisre** [comment]
I am still experiencing this. I have a feeling that this is some permissions issue. What can I do to help get to the bottom of this?

**Comment by fabykr** [comment]
Thanks for your work! I couldn't import a recipe export from Nextcloud Cookbook either. Error message "failure".

The last thing in the debug log is:

"wsgi.errors:<gunicorn.http.wsgi.WSGIErrorsWrapper object at 0x7fa8b60a5700>
wsgi.version:(1, 0)
wsgi.multithread:False
wsgi.multiprocess:False
wsgi.run_once:False
wsgi.file_wrapper:
wsgi.input_terminated:True
wsgi.input:<gunicorn.http.body.Body object at 0x7fa8b609f1f0>
wsgi.url_scheme:https"

**Comment by navanchauhan** [comment]
Yep, I am aware of this issue and am investigating this along with #35 now that I have time

Best,

Navan

**Comment by Thovi98** [comment]
Does it work in latest 2.0.1 version?

**Comment by Thovi98** [comment]
Feel free to reopen if you encounter that again and provide the needed logs :)
